feature-pack/src/main/resources/modules/system/layers/base/org/jboss/jts/main/module.xml

```diff
@@ -47,5 +47,11 @@
         <module name="org.jboss.weld.core"/>
         <module name="javax.annotation.api" export="true" />
         <module name="javax.interceptor.api" export="true" />
+        <module name="org.wildfly.transaction.client"/>
+        <module name="org.jboss.as.transactions">
+            <imports>
+                <include path="org.jboss.as.txn.integration"/>
```

Comment:
unless it causes an issue to get this merged upstream I think mixing "." and "/" in the same file would be best avoided, but we could do a follow up if there is a good reason to use "/"